charts/icm-as/tests/default-values_test.yaml

```diff
@@ -89,7 +89,7 @@ tests:
           value: 1000m
       - equal:
           path: spec.template.spec.containers[0].resources.limits.memory
-          value: 3Gi
+          value: 6Gi
```

Comment:
If you want to avoid, that a pod is killed/moved, you have to define an according PodDisruptionBudget: https://kubernetes.io/docs/tasks/run-application/configure-pdb/

Reply:
But there is another thing, that's true. memory.limits and memory.requests should be identical to avoid killing pods. The explanation is simple, memory.requests will be used to select the host were to start the pod. If the pod grows over memory.requests and the host is not able to provide the memory, the pod will be killed and restarted. Unfortunately very likely on the same host, which provides sufficient memory for memory.requests but not for memory.limits.